File: Python/1158 Soma de Ímpares Consecutivos III.py
n = int(input())
x = 0
for i in range(1, n+1):
    s = list(input().split())
    a = int(s[0])
    b = int(s[1])
    for j in range(1,b+1):
        if a%2 != 0: x+=a
        else: 
            a+=1
            x+=a
        a+=2
    print(x)
    x = 0

File: C#/1158 Soma de Ímpares Consecutivos III.cs
using System;

class MainClass {
  public static void Main (string[] args) 
  {
    Console.Clear();
    int x = 0;
    int n = int.Parse(Console.ReadLine());
    for(int i = 1; i <= n; i++)
    {
      string[] s = Console.ReadLine().Split(' ');
      int a = int.Parse(s[0]);
      int b = int.Parse(s[1]);
      for(int j = 1; j <= b; j++)
      {
        if(a % 2 != 0) x += a;
        else { a++; x += a;}
        a += 2;
      }
      Console.WriteLine(x);
      x = 0;
    }
  }
}
